app/models/patient.rb

```diff
@@ -8,6 +8,12 @@ class Patient < ApplicationRecord
   has_many :phone_numbers, class_name: 'PatientPhoneNumber'
   has_many :blood_pressures
 
+  has_many :facility_patients
+  has_many :facilities, through: :facility_patients
+
+  validates :device_created_at, presence: true
+  validates :device_updated_at, presence: true
+
```

Comment:
Where is the facility assigned to a patient? Should that be a part of the patient sync API?